### PR TITLE
update taproot faq

### PR DESCRIPTION
**Pull request by MarnixCroes**
At the time, I'm quite sure that someone said that change is always to a taproot address, but it's not.

was probably a misunderstanding or miscommunication.

anyway, fix it here

### CODE REVIEW

**Review by yahiheb — COMMENTED** [review]
I am not sure either, so I let @lontivero decide on this one.

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ It also shows some information about the current coinjoin round.
 Partially, Wasabi supports:
 - sending to Taproot (bech32m) address format
 - receiving Taproot outputs from a coinjoin
-- change from a normal transaction is always received to a Taproot address
+- receiving Taproot change from a normal transaction
```
Comment (lontivero) [inline docs/FAQ/FAQ-UseWasabi.md:241]: I've read this 10 times and have no idea what it means.

Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md]: changed it

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ It also shows some information about the current coinjoin round.
 Partially, Wasabi supports:
 - sending to Taproot (bech32m) address format
 - receiving Taproot outputs from a coinjoin
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:240]: As Lucas said I guess this one also should be changed.

Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:240]: I understood it as that (only) line 241 didn't make sense

**Comment by lontivero** [comment]
I think we should define what we mean buy support taproot. In the UI there is nothing yes. Internally Wasabi uses taproot for some outputs in the coinjoins (this means that the amount decomposer takes the script types into account) and for change outputs in normal transactions almost randomly too.
In the coin selector IDK but in the RPC we can see the taproot scripts/addresses, we can generate new taproot addresses (so, we can receive money from others to those taprrot addresses) , etc. We can pay in coinjoin to a taproot script. We generate compact filters with taproot scripts in order to recognize those blocks containing transactions from/to our wallet, etc,etc

**Review by yahiheb — COMMENTED** [review]
I would simplify this by saying:

Wasabi supports taproot.
- Sending to taproot addresses is supported.
- Generating taproot addresses is supported using the RPC but not in the UI.

A note can be added explaining more how this works for coinoins.

**Comment by MarnixCroes** [comment]
simplified it by saying Wasabi supports Taproot, except for address generation in GUI.

and put a small note to mention possibility of having taproot coins.

**Comment by MarnixCroes** [comment]
> Generating taproot addresses is supported using the RPC but not in the UI.

the `getnewaddress` RPC doesn't return taproot, so this can be confusing

**Comment by lontivero** [comment]
Sorry but I think it is still confusing. I would say that Wasabi supports taproot except for that fact that it doesn't allow you to create taproot addresses. I you don't agree then merge it as it is.

**Comment by MarnixCroes** [comment]
> Sorry but I think it is still confusing. I would say that Wasabi supports taproot except for that fact that it doesn't allow you to create taproot addresses. I you don't agree then merge it as it is.

ok done: _Wasabi supports Taproot, except that the user cannot generate Taproot addresses._

**Review by yahiheb — APPROVED** [review]

**Pull request merged**